src/sys/signal.rs

```diff
@@ -831,6 +831,95 @@ mod sigevent {
     }
 }
 
+/// A wrapper for `stack_t`, used to represent alternate stacks for `sigaltstack`.
+#[derive(Copy, Clone)]
+#[allow(missing_debug_implementations)]
+pub struct SigStack {
+    stack: libc::stack_t,
+}
+
+impl SigStack {
+    /// Define an alternate signal stack for use with `sigaction`.
+    ///
+    /// See [`sigaltstack(3posix)`](http://man7.org/linux/man-pages/man3/sigaltstack.3p.html) for
```

Comment:
For standardized functions, link to the open group's man pages instead of Linux's.  In this case: http://pubs.opengroup.org/onlinepubs/9699919799/ .